manuscript/draft.Rmd

```diff
@@ -166,3 +177,84 @@ None of the models are true forcasts as they use observed predictor variables (e
 * Need predictions at multiple time scales
 * Prediction quality at short time scales might not be a good indicator of 
   long-term effectiveness
+
+
+
+## Box 1: Ten simple rules for making and evaluating ecological forecasts
```

Comment:
We're missing spatial/site level stuff, I think.  I think something like 30% of the variance in species richness was site-level stuff that couldn't be explained by annual fluctuations, observer effects, or the environmental variables we included.  This would be a good place to talk about mixed models, latent variables, JSDMs, spatial random effects, etc.